src/Bravo3/Orm/Drivers/Redis/RedisDriver.php

```diff
@@ -11,19 +11,28 @@
 use Bravo3\Orm\KeySchemes\StandardKeyScheme;
 use Bravo3\Orm\Services\ScoreNormaliser;
 use Bravo3\Orm\Traits\DebugTrait;
-use Predis\Client;
 use Predis\Command\CommandInterface;
+use Predis\ClientInterface;
 use Predis\Command\KeyScan;
 use Predis\Connection\Aggregate\PredisCluster;
 use Predis\Connection\Aggregate\ReplicationInterface;
 use Predis\Connection\NodeConnectionInterface;
 use Predis\Pipeline\Pipeline;
+use Predis\Response\ServerException;
+use Predis\Connection\ConnectionException;
 
 class RedisDriver implements DriverInterface
 {
     use DebugTrait;
 
     /**
+     * Get/Write commands that fails will be retried again
+     * upto $max_connection_retries times with delays between
+     * each retry.
+     */
+    const RETRY_DELAY_MULTIPLIER = 1.5;
```

Comment:
Yeah, it feels like something that might come in handy one day. Just make it a protected var and offer a getter & setter